docs/API.md

```diff
@@ -668,6 +669,70 @@ except ResponseError as err:
     print(err)
 ```
 
+<a name="select_object_content"></a>
+### select_object_content(self, bucket_name, object_name, obj)
+Select object content filters the contents of object based on a simple structured query language (SQL).
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|:---|:---|:---|
+|``bucket_name``   |_string_   |Name of the bucket.   |
+|``object_name``   |_string_   |Name of the object.   |
+|``obj`` | _Object_ | Query Options   |
+
+
+__Return Value__
+
+|Param   |Type   |Description   |
+|:---|:---|:---|
+|``object``   | _urllib3.response.HTTPResponse_   |Represents http streaming reader.   |
```

Comment:
SelectObjectReader should be listed here not HTTPResponse

Reply:
done.